Read-and-Save-document/Open-and-save-Word-document/Google-Drive/Open-Word-document/Open-Word-document/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Google.Apis.Auth.OAuth2;
+using Google.Apis.Drive.v3;
+using Google.Apis.Services;
+using Google.Apis.Util.Store;
+
+namespace Open_Word_document
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            UserCredential credential;
+            string[] Scopes = { DriveService.Scope.DriveReadonly };
+            string ApplicationName = "YourAppName";
+            // Step 1: Open Google Drive with credentials.
+            using (var stream1 = new FileStream("credentials.json", FileMode.Open, FileAccess.Read))
+            {
+                string credPath = "token.json";
+                credential = GoogleWebAuthorizationBroker.AuthorizeAsync(
+                    GoogleClientSecrets.Load(stream1).Secrets,
+                    Scopes,
+                    "users",
+                    CancellationToken.None,
+                    new FileDataStore(credPath, true)).Result;
+            }
+
+            // Step 2: Create Drive API service
+            var service = new DriveService(new BaseClientService.Initializer()
```

Comment:
Add pull stop at the end of the comment.

Reply:
Added properly.